var/spack/repos/builtin/packages/linux-pam/package.py

```diff
@@ -10,8 +10,12 @@ class LinuxPam(AutotoolsPackage):
     """Linux PAM (Pluggable Authentication Modules for Linux) project."""
 
     homepage = "http://www.linux-pam.org/"
-    url      = "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-1.3.1.tar.xz"
+    url      = "https://github.com/linux-pam/linux-pam/releases/download/v1.5.2/Linux-PAM-1.5.2.tar.xz"
```

Comment:
The URL doesn't need to be updated since it just serves as a template for spack's version parsing. The new value is fine though.

Reply:
Hi @sethrj

I see, I did not realize that. Then do you think we can leave as it is right now?

Thanks

Reply:
Yep, it's good as is! We can let the automerge keep going.